### PR TITLE
fix cpo jobs

### PR DESCRIPTION
**Pull request by adisky**
go get kubetest in not installing succesfully from
 $GOPATH/k8s.io/src/cloud-provider-openstack
need to run it from non repo

### CODE REVIEW

File: playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml [hunk 1]
```diff
@@ -116,7 +117,7 @@
 
           # Run e2e test using local deployment/provider
           # Must run kubetest under kubernetes root directory
-          cd '{{ k8s_src_dir }}'
+          pushd '{{ k8s_src_dir }}'
```
Comment (ramineni) [inline playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml]: no need to change here I suppose

File: playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml [hunk 2]
```diff
@@ -103,6 +103,7 @@
           export KUBECONFIG=/var/run/kubernetes/admin.kubeconfig
 
           # Build kubetest
+          cd ..
```
Comment (ramineni) [inline playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml]: May be add comment for future reference, your call ?

Comment (wangxiyuan) [inline playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml]: it's good to add a comment that why `cd` here.

File: playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml (hunk 1 above)
Comment (wangxiyuan) [inline playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml]: single using `pushd` is useless. just `cd` here is OK.

**Pull request merged**